assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/filled-mini-cart-contents-block/frontend.tsx

```diff
@@ -3,18 +3,22 @@
  */
 import { useStoreCart } from '@woocommerce/base-context/hooks';
 
+type FilledMiniCartContentsBlockProps = {
+	children: JSX.Element;
+	className: string;
+};
+
 const FilledMiniCartContentsBlock = ( {
 	children,
-}: {
-	children: JSX.Element | JSX.Element[];
-} ): JSX.Element | null => {
+	className,
+}: FilledMiniCartContentsBlockProps ): JSX.Element | null => {
 	const { cartItems } = useStoreCart();
 
 	if ( cartItems.length === 0 ) {
 		return null;
 	}
 
-	return <>{ children }</>;
+	return <div className={ className }>{ children }</div>;
```

Comment:
For blocks that didn't have any kind of markup like this one, I wonder if instead of adding support for custom class names, maybe we should just set [`supports: { customClassName: false }`](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#customclassname). If users want to add custom class names to be able to add styles, they can still add them to the block which render some markup. What do you think?

Reply:
Mmmm, good point!


The problem that I see is that the HTML (attached image) generated is different from [the declared template](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/87ccbddfa06bcd5bd0d7fd5398bf201ac6705c55/templates/parts/mini-cart.html#L4). From my experience it seems that every block should have a wrapper.

![image](https://user-images.githubusercontent.com/4463174/156776733-a1727528-6c2a-4eca-b83a-2d92d766b5ce.png).

Reply:
> If users want to add custom class names to be able to add styles, they can still add them to the block which render some markup.

This approach sounds good to me too. Filled and Empty Mini Cart Contents blocks are always inside the Mini Cart Contents block. Users can add custom classes there to target inner blocks.

Reply:
> The problem that I see is that the HTML (attached image) generated is different from [the declared template](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/87ccbddfa06bcd5bd0d7fd5398bf201ac6705c55/templates/parts/mini-cart.html#L4). From my experience it seems that every block should have a wrapper.

What about this? I guess that now, the Gutenberg team is [adding more APIs for container block](https://github.com/WordPress/gutenberg/issues/27324), I prefer to keep this the wrapper in this way, we:
- are consistent with the template.
- are future proof, since if we want to add support for the new API from Gutenberg we already have the wrapper (I guess no migration).
- follow the standard (I'm not sure about this point, but as I said above, it seems that every block has a dedicated wrapper).

Reply:
Good points. Honestly, I don't have a strong opinion on this, but agree that it makes sense keeping it consistent with other blocks that also have a wrapper div and allow a custom class name. So :shipit: from my side.